test/test_distributions.py

```diff
@@ -51,6 +52,10 @@
 )
 from numpyro.nn import AutoregressiveNN
 
+# Enable 64bit support for higher accuracy
+if sys.maxsize > 2**32:
+    jax.config.update("jax_enable_x64", True)
```

Comment:
Could you revert this change? You can cover the failing tests by adding this line at the top of the test
```python
if jax_dist in [LowerTruncatedPowerLaw, DoublyTruncatedPowerLaw]:
    numpyro.enable_x64()
```

Reply:
Resolved in 96d5f6c